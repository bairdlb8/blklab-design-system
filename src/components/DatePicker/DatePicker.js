import React from 'react'
import PropTypes from 'prop-types'

import './DatePicker.scss'

const DatePicker = ({className=''}) => {
  return <div>DatePicker</div>
}

DatePicker.propTypes = {}

export default DatePicker
