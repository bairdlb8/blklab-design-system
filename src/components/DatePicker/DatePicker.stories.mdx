import { Meta, Story, Props, Preview } from '@storybook/addon-docs/blocks'

import DatePicker from './DatePicker'

<Meta title="Components / DatePicker" component={ DatePicker } />

# DatePicker

## Add some text here

export const Template = (args) => {
  return <DatePicker {...args} />
}

<Preview withToolbar>
  <Story name="Primary" args={ {} }>
    {Template.bind({})}
  </Story>
</Preview>

<Props of={ DatePicker } />
