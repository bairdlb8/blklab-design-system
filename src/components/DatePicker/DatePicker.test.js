import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { axe, toHaveNoViolations } from 'jest-axe'
import DatePicker from './DatePicker'

expect.extend(toHaveNoViolations)

describe('<DatePicker />', () => {
  describe('accessibility', () => {
    it('should not fail any accessibility tests', async () => {
      const { container } = render(<DatePicker />)
      expect(await axe(container)).toHaveNoViolations()
    })
  })

  it('should take a snapshot', () => {
    const { asFragment } = render(<DatePicker />)

    expect(asFragment(<DatePicker />)).toMatchSnapshot()
  })
})
