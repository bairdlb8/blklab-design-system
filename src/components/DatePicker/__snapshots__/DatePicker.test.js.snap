// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DatePicker /> should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    DatePicker
  </div>
</DocumentFragment>
`;
